src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -732,6 +732,17 @@ public ConsoleColorProxy(ConsoleHostUserInterface ui)
                 _ui = ui;
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Import-PSSession Cmdlet error handling.Test content and format of proxied error message (Windows 7: #319080).Test non-terminating error
```powershell
Expected $true, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Implicit.Remoting.Tests.ps1: line 209
209:                 ($errorString -like "*VariableNotFound*") | Should -BeTrue
```